plugins/chat/plugin.rb

```diff
@@ -271,7 +271,7 @@ def self.onebox_template
         next if !chat_channel
       end
 
-      next if !Guardian.new.can_see_chat_channel?(chat_channel)
+      next if !Guardian.new.can_preview_chat_channel?(chat_channel)
```

Comment:
same question about tests

Reply:
This method was renamed and onebox serializer was tested here https://github.com/discourse/discourse/blob/68c4f16a730931bb37d79520c3636a689c302006/plugins/chat/spec/plugin_spec.rb#L169